### PR TITLE
User scope check

### PR DESCRIPTION
**Pull request by valentinsulzer**
Implement user scope checking

### CODE REVIEW

File: ionwizard/library_wizard.py [hunk 1]
```diff
@@ -26,8 +26,6 @@ def install_from(config):
             addr = IonWorksPipWizard.get_address(library["key"])
             if library["install"]:
                 IonWorksPipWizard.install_library(library["library"], addr)
-            else:
-                print(f'\n{library["library"]} --index-url {addr}\n')
```
Comment (kratman) [inline ionwizard/library_wizard.py:30]: This was to help with creating pip.conf files and things like requirements files or poetry which need index URLs.

If we are removing the "install: false" functionality then we needs some extra functions and update the docs

File: ionwizard/validate.py [hunk 2]
```diff
@@ -91,3 +97,7 @@ def machine_id_check():
     if config.get("machine_id") != machine_id:
         return {"success": False, "message": "Error: Machine ID mismatch"}
     return {"success": True, "message": "Machine ID is valid"}
+
+
+if __name__ == "__main__":
+    print(license_check("ionworkspipeline"))
```
Comment (kratman) [inline ionwizard/validate.py:103]: What do we need this part for?

Comment (valentinsulzer) [reply ionwizard/validate.py:103]: forgot to remove this

File: ionwizard/library_wizard.py (hunk 1 above)
Comment (valentinsulzer) [reply ionwizard/library_wizard.py:30]: It's getting printed without any explanation right now so not that useful to the user - it just makes it look like we forgot to remove a debugging print statement

Comment (kratman) [reply ionwizard/library_wizard.py:30]: I removed the docs for that feature. I will have to put it back in for later since it is useful

**Review by valentinsulzer — COMMENTED** [review]
Updates look good to me

**Review by kratman — APPROVED** [review]

**Pull request merged**